### ISSUE TITLE
Wont get any informations 

### ISSUE THREAD

**Issue by unclesam87** [Yogibaer75/Check_MK-Things#22]
hey,
have an weird problem, some of my checkm instances are able to pick up informations, some wont show anything
when runniny manual over shell i get the following errors:
```
Traceback (most recent call last):
  File "/opt/omd/sites/fpi/local/share/check_mk/agents/special/./agent_ilo", line 347, in <module>
    get_information(REDFISHOBJ)
  File "/opt/omd/sites/fpi/local/share/check_mk/agents/special/./agent_ilo", line 94, in get_information
    ilogen, iloversion, prefix, res_dir = get_gen(redfishobj)
  File "/opt/omd/sites/fpi/local/share/check_mk/agents/special/./agent_ilo", line 83, in get_gen
    if ilogen.split(' ')[-1] == "CM":
AttributeError: 'NoneType' object has no attribute 'split'
```
Any ideas?

**Comment by Yogibaer75** [comment]
Can you please check the used version of the iLO plugin?
The actual one is --> hpe_ilo-3.7.mkp 
This error should be fixed with version 3.7

**Comment by unclesam87** [comment]
thx works fine!
